examples/asr/emformer_rnnt/pipeline_demo.py

```diff
@@ -67,15 +67,15 @@ def run_eval_streaming(args):
                 features, length = streaming_feature_extractor(segment)
                 hypos, state = decoder.infer(features, length, 10, state=state, hypothesis=hypothesis)
             hypothesis = hypos[0]
-            transcript = token_processor(hypothesis.tokens, lstrip=False)
```

Comment:
Does libtorch-light support `namedtuple`? `hypothesis.tokens` is more intuitive to users than `hypothesis[0]`